gulp/test.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+var gulp = require('gulp'),
+    jasmineNode = require('gulp-jasmine-node');
+
+/**
+ * Set the env
+ */
+gulp.task('env:test', function() {
+  process.env.NODE_ENV = 'test';
+});
+
+/**
+ * Run Jasmine tests
+ */
+gulp.task('runJasmine', ['env:test'], function() {
+  return gulp.src(['test/*spec.js'])
```

Comment:
SettingJasmine is to run anything post-fixed with `spec,js`, as there will eventually be some setup files in the test folder that aren't necessarily needed for each test run.